src/m365/spo/commands/list/list-add.ts

```diff
@@ -658,9 +658,7 @@ class SpoListAddCommand extends SpoCommand {
       requestBody.DisableGridEditing = options.disableGridEditing;
     }
 
-    if (options.draftVersionVisibility) {
-      requestBody.DraftVersionVisibility = options.draftVersionVisibility;
-    }
+    requestBody.DraftVersionVisibility = options.draftVersionVisibility ? DraftVisibilityType[(options.draftVersionVisibility.trim() as keyof typeof DraftVisibilityType)] : DraftVisibilityType.Reader;
```

Comment:
Since we removed the default value in the docs (which is good in my opinion), we shouldn't also use a default value here. Let's just set the value when the option is set.